src/_interfaces/injector.js

```diff
@@ -0,0 +1,26 @@
+// @flow
```

Comment:
This tells flow to type check a particular file.  Flow doesn't look at files that don't have "// @flow" We could move all declared interfaces to a single file, but this seems more organized, and in line with what other projects i've looked at do.